docs/index.rst

```diff
@@ -0,0 +1,10 @@
+
+Contents
```

Comment:
I think for now it is ok, but at some point it would be necessary to provide a bit more context/use cases for that package. You can check [peakdet doc](https://peakdet.readthedocs.io/en/latest/index.html) for an example